docs/book/src/introduction/sway_quickstart.md

```diff
@@ -202,6 +202,26 @@ Note the contract ID — you will need it if you want to build out a frontend to
 
 If `forc-deploy` emits a `no such subcommand` error, try to install it with [fuelup](../introduction/installation.md) or [manually](https://github.com/FuelLabs/sway/releases).
 
+## Deploying the contract with Salt
+
+You can pass a Salt using the `--salt` flag in your `forc deploy` command to redeploy a contract that has been deployed already. This is useful for multiple deployments of the same contract.
+
+To add a Salt to your ContractID while deployment, you can run the following command:
+
+```bash
+forc deploy --salt <Salt> --node-url <key> --gas-price 1
+```
+
+### Randomised Salt
+
+Salt is 0x00000000000000000000000000000000 by default. If you do not want to pass a salt manually, you can also use the --random-salt flag to randomise your salt as follows:
+
+```bash
+forc deploy --random-salt --node-url <key> --gas-price 1
```

Comment:
We seem to be deploying with `--unsigned` in this document and none of the other flag. Maybe we should follow that convention here as well, as in:
```
forc deploy --unsigned --random-salt
```
That way, we are _building_ on what's already been explained before.